recipe/meta.yaml

```diff
@@ -8,19 +8,15 @@ package:
   version: {{ version }}
 
 source:
-  - url: https://web.archive.org/web/{{ version }}/https://netlib.sandia.gov/f2c/libf2c.zip
+  - url: https://www.netlib.org/{{ name }}/libf2c.zip
```

Comment:
I saw the discussion around web.archive.org vs netlib, but this URL doesn't appear to have a version. Why do we not need a version-specific download?

Reply:
Not sure on that either

Reply:
I recommend keeping the archive.org URL. For reproducibility and auditability reasons, we need to be able to tie each package build to a specific, versioned source package; and unfortunately, netlib does not seem to have versioned releases.

Reply:
@chenghlee Could we host the artifacts ourselves instead of relying on the internet archive?

Reply:
ok